packages/web3-eth-accounts/src/index.js

```diff
@@ -81,7 +81,17 @@ var Accounts = function Accounts() {
             }, function() {
                 return 'latest';
             }]
-        })
+        }),
+        new Method({
+            name: 'getBlockByNumber',
+            call: 'eth_getBlockByNumber',
+            params: 2,
+            inputFormatter: [function(blockNumber) {
+                return blockNumber ? utils.toHex(blockNumber) : 'latest'
+            }, function() {
+                return false
+            }]
+        }),
```

Comment:
is this a new method?

Reply:
Sorta, this package doesn't make use of the RPC wrappers found in `web3-eth`, instead needed methods are redefined in the package. I added this method to be able to get a block to find the `baseFee`

Reply:
You mean we can't leverage the [getBlock](https://github.com/ChainSafe/web3.js/blob/v1.4.0/packages/web3-eth/src/index.js#L420-L426) (also refs [blockCall](https://github.com/ChainSafe/web3.js/blob/v1.4.0/packages/web3-eth/src/index.js#L44-L46)) because it's defined in the `web3-eth` package and not accessible from `web3-eth-accounts`?
In this case don't we have a dependency or at least a DRY issue that could be addressed somehow?

Reply:
@AndreMiras Yes, and we plan to address the problematic architecture in `4.x` (our rewrite), so for now it's just about making the least amount of change to `1.x` as possible, to refrain from introducing additional bugs